### DISCUSSION TITLE
resume after power loss E3V2

### DISCUSSION THREAD

**Discussion by bradaz11** [Jyers/Marlin#1836] [General Discussion]
hi, I'm using the precompiled firmware, E3V2-BLTouch-3x3-HS-v4.2.2-v2.0.1

I turned on the resume after power loss function, but when it did this, it checks limits and reprobes the centre before resuming the print. obviously if I had a print half done, it would crash into the print. How do I stop it from doing this? and is it meant to?

also, could the unload and reload values for the filament be preloaded on the software? I'm assuming the value should be distance in mm?

**Comment by Cal-LF** [comment]
@bradaz11 The load and unload lengths are unique to each printer. Determined by length of feed tube and any other lengths in the feed path. You can enter your printer's feed lengths in the config.h file and compile it. That is what I do with many of the settings.
